package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@emotion/core": "^10.1.1",
     "@emotion/styled": "^10.0.27",
-    "@mdx-js/mdx": "^1.6.19",
+    "@mdx-js/mdx": "^1.6.22",
```

Comment:
We generally tend to update all mdx packages at once (when possible). Do you mind reverting this for now?